pkg/flag/kubernetes_flags.go

```diff
@@ -160,14 +149,7 @@ func (f *K8sFlagGroup) ToOptions() (K8sOptions, error) {
 	if err != nil {
 		return K8sOptions{}, err
 	}
-	var parallel int
-	if f.Parallel != nil {
-		parallel = getInt(f.Parallel)
-		// check parallel flag is a valid number between 1-20
-		if parallel < 1 || parallel > 20 {
```

Comment:
Perhaps we need to keep these (or other) restrictions for new scan flag:
https://github.com/aquasecurity/trivy/blob/c8eaa4c0f961c67a675d0d35d20171c1d58de257/pkg/flag/scan_flags.go#L62-L67

Reply:
I give a special meaning to `--parallel 0`.
https://github.com/aquasecurity/trivy/pull/5572/commits/c08df01eb27b4f2189823175785b18233bc9c8dc

I'm not sure about the maximum. Recent computers can have many CPU cores and enough bandwidth.

Reply:
okay, good. 
Can you fix test?

Reply:
fixed